src/utils/roles.utils.js

```diff
@@ -1,8 +1,3 @@
-const roles = {
-	SUPER_ADMIN: '94339a6a-2fc1-4bfc-814c-616b72c87306',
-	TRAVEL_ADMIN: 'f24659cc-e9bd-43ee-b97a-294b3c154a78',
-	MANAGER: '69263f66-8890-419a-97e7-b9f2f1e5e578',
-	REQUESTER: '0a422b6d-d6bc-4373-ba91-c51268323fc1',
-};
+const roles = ['SUPER_ADMIN','TRAVEL_ADMIN','MANAGER','REQUESTER',];
 
 module.exports = roles;
```

Comment:
Use ES6

Reply:
The roles and the hashPassword functions are being used in models and seeders files which when running with sequelize they do not use babel so thereby requiring to use es5 files even in import